client/components/date-range/style.scss

```diff
@@ -110,19 +110,25 @@ $date-range-mobile-layout-switch: $break-small;
 	justify-content: flex-end;
 	padding: 10px 0 0;
 
-	.button {
+	.components-button {
 		margin-left: 0.5em;
 		margin-right: 0;
+		&.is-compact {
+			height: 28px;
+			font-size: $font-body-extra-small;
+		}
 	}
 	margin-top: 10px;
 
 	@media (max-width: $date-range-mobile-layout-switch) {
 		flex-direction: column-reverse;
 		align-items: stretch;
 
-		.button {
+		.components-button {
 			height: 40px;
```

Comment:
This `height` is overridden by the `is-compact` height, so it should also be removable.

Reply:
Addressed in de5ffed3613